az-core/src/main/java/azkaban/Constants.java

```diff
@@ -325,6 +326,10 @@
     public static final String AZKABAN_RAMP_STATUS_POLLING_INTERVAL = "azkaban.ramp.status.polling.interval";
     public static final String AZKABAN_RAMP_STATUS_POLLING_CPU_MAX = "azkaban.ramp.status.polling.cpu.max";
     public static final String AZKABAN_RAMP_STATUS_POLLING_MEMORY_MIN = "azkaban.ramp.status.polling.memory.min";
+
+    public static final String EXECUTION_LOGS_RETENTION_MS = "execution.logs.retention.ms";
+    public static final String EXECUTION_LOGS_CLEANUP_INTERVAL_SECONDS = "execution.logs.cleanup"
```

Comment:
Thanks for making these string constants, but please try not to break strings like that -- that makes them unsearchable or at least less findable. Specifically, if I saw `execution.log.cleanup.interval.seconds` in a config file and wanted to figure out how it is used in code, I would not be able to (easily) find any occurrences.

Reply:
I see your point. Right now if I have to find the occurrence of a key in config it can be a two-step work. 1. Find the const that maps to this string 2. the find the uses of that const. Because I had to use the same const in the test file also I defined it here so that it's not duplicated.